### PR TITLE
Bring accentued letters together on company database page II #642

### PR DESCRIPTION
**Pull request by Divyateja04**
As requested in the issue #642 , the accentued letters were added to the database page
Please check, incase you feel something is wrong, please let me know and I'll be glad to fix it

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -26,3 +26,4 @@ Rafael Audibert
 Steffen Grundke
 WL
 wolffshots
+Divyateja04
```
Comment (mal-tee) [inline AUTHORS:29]: I think this should be sorted.

Comment (Divyateja04) [reply AUTHORS:29]: means I shouldn't have added my name?

Comment (Divyateja04) [reply AUTHORS:29]: like I saw that your bot was asking so I added before itself, so I thought I'll add

Comment (baltpeter) [reply AUTHORS:29]: No, you can absolutely add your name. But the list should be sorted alphabetically.

Comment (Divyateja04) [reply AUTHORS:29]: oh like that...yeah I'll do that

Comment (Divyateja04) [reply AUTHORS]: yeah done, can you check the PR?

**Review by mal-tee — COMMENTED** [review]
Looks good. But it seems like the scrollspy broke. Not sure if this is a bug in your PR o.0. Pinging @baltpeter and @zner0L

**Comment by mal-tee** [comment]
image
I.e. clicking on these letters doesn't automatically scroll down. And scrolling doesn't highlight the current letter.

File: layouts/company/list.html [hunk 2]
```diff
@@ -17,15 +17,32 @@
...
+        {{ $clean_name := replaceRE "[ÅÁÀÂÄÃÅÆΆ]" "A" (upper .Params.name) }}
+        {{ $clean_name := replaceRE "[B̌B́]" "B" $clean_name }}
+        {{ $clean_name := replaceRE "[ÇČĆ̣Ḉ]" "C" $clean_name }}
+        {{ $clean_name := replaceRE "[ĎÐD́]" "D" $clean_name }}
+        {{ $clean_name := replaceRE "[ÉÈÊËĚÊ̌ΕĖ́Ę́É̩Έ]" "E" $clean_name }}
+        {{ $clean_name := replaceRE "[F̌F́]" "F" $clean_name }}
+        {{ $clean_name := replaceRE "[ǦǴ]" "G" $clean_name }}
+        {{ $clean_name := replaceRE "[ȞH́Ή]" "H" $clean_name }}
+        {{ $clean_name := replaceRE "[ÌÍÎÏΊ]" "I" $clean_name }}
+        {{ $clean_name := replaceRE "[J̌J́]" "j" $clean_name }}
```
[the remark this answers is not in the document]
Comment (mal-tee) [reply layouts/company/list.html:29]: This might be the issue?

[the remark this answers is not in the document]
Comment (Divyateja04) [reply layouts/company/list.html:29]: but that's only j, remaining should still work then right?
lemme check by running it locally once by changing j to J

[the remark this answers is not in the document]
Comment (mal-tee) [reply layouts/company/list.html:29]: While messing around in the deploy preview it looked like the spy stopped working at J. :D

File: AUTHORS [hunk 3]
```diff
@@ -25,4 +26,4 @@ Phoenix Déb
 Rafael Audibert
 Steffen Grundke
 WL
-wolffshots
+wolffshots
```
Comment (mal-tee) [inline AUTHORS:29]: We want a newline at the end of this file. Besides that we are ready to merge!

File: layouts/company/list.html [hunk 4]
```diff
@@ -17,15 +17,32 @@
...
+        {{ $clean_name := replaceRE "[ŘŘ̩Ŕ]" "R" $clean_name }}
         {{ $clean_name := replaceRE "[ŚṠŜŠŞ]" "S" $clean_name }}
-        {{ $clean_name := replaceRE "[ÝŸ]" "Y" $clean_name }}
+        {{ $clean_name := replaceRE "[ŤT́]" "T" $clean_name }}
+        {{ $clean_name := replaceRE "[ŬÙÚÛÜ]" "U" $clean_name }}
+        {{ $clean_name := replaceRE "[V̌V́]" "V" $clean_name }}
+        {{ $clean_name := replaceRE "[W̌Ẃ]" "W" $clean_name }}
+        {{ $clean_name := replaceRE "[X̌X́Ȳ́]" "X" $clean_name }}
+        {{ $clean_name := replaceRE "[ÝŸΎ]" "Y" $clean_name }}
+        {{ $clean_name := replaceRE "[ŽŹ]" "Z" $clean_name }}
```
Comment (mal-tee) [inline layouts/company/list.html:45]: Did you research for other variations as well? I think we are missing some chars from the greek alphabet.

Comment (Divyateja04) [reply layouts/company/list.html:45]: ohh I added most I could find, can you tell me which type of greek alphabet we are missing out?

File: AUTHORS (hunk 3 above)
Comment (Divyateja04) [reply AUTHORS:29]: oh okay I'll add that too

Comment (Divyateja04) [reply AUTHORS]: Done with this

File: layouts/company/list.html [hunk 5]
```diff
@@ -17,15 +17,32 @@
...
-        {{ $clean_name := replaceRE "[ÉÈÊË]" "E" $clean_name }}
-        {{ $clean_name := replaceRE "[ÌÍÎÏ]" "I" $clean_name }}
-        {{ $clean_name := replaceRE "[ÒÓÔÕÖØ]" "O" $clean_name }}
-        {{ $clean_name := replaceRE "[ŬÙÚÛÜ]" "U" $clean_name }}
-        {{ $clean_name := replaceRE "[Ñ]" "N" $clean_name }}
+        {{ $clean_name := replaceRE "[ÅÁÀÂÄÃÅÆΆ]" "A" (upper .Params.name) }}
+        {{ $clean_name := replaceRE "[B̌B́]" "B" $clean_name }}
+        {{ $clean_name := replaceRE "[ÇČĆ̣Ḉ]" "C" $clean_name }}
+        {{ $clean_name := replaceRE "[ĎÐD́]" "D" $clean_name }}
+        {{ $clean_name := replaceRE "[ÉÈÊËĚÊ̌ΕĖ́Ę́É̩Έ]" "E" $clean_name }}
```
Comment (mal-tee) [inline layouts/company/list.html:24]:
```suggestion
        {{ $clean_name := replaceRE "[ÉÈÊËĚÊ̌ΕĖ́Ę́É̩]" "E" $clean_name }}
```
The capital epsilon was duplicated. Or did I miss sth?

File: layouts/company/list.html (hunk 4 above)
Comment (mal-tee) [reply layouts/company/list.html:45]: - Capital Beta: Β
- Capital Zeta: Ζ
- Capital Kappa: Κ
- ...

Check e.g. https://en.wikipedia.org/wiki/Greek_alphabet

File: layouts/company/list.html (hunk 5 above)
Comment (Divyateja04) [reply layouts/company/list.html:24]: yeah last one is not epsilon, it's Έ

Comment (mal-tee) [reply layouts/company/list.html:24]: oops, my bad. To be precise, seems like its both: https://www.fileformat.info/info/unicode/char/0388/index.htm But we want to keep it. :)

File: layouts/company/list.html [hunk 6]
```diff
@@ -18,31 +18,31 @@
              alphabet characters.
         */}}
         {{ $clean_name := replaceRE "[ÅÁÀÂÄÃÅÆΆ]" "A" (upper .Params.name) }}
-        {{ $clean_name := replaceRE "[B̌B́]" "B" $clean_name }}
+        {{ $clean_name := replaceRE "[B̌B́Β]" "B" $clean_name }}
         {{ $clean_name := replaceRE "[ÇČĆ̣Ḉ]" "C" $clean_name }}
-        {{ $clean_name := replaceRE "[ĎÐD́]" "D" $clean_name }}
+        {{ $clean_name := replaceRE "[ĎÐD́Δ]" "D" $clean_name }}
```
Comment (mal-tee) [inline layouts/company/list.html:23]:
```suggestion
        {{ $clean_name := replaceRE "[ĎÐD́]" "D" $clean_name }}
```

File: layouts/company/list.html [hunk 7]
```diff
@@ -18,31 +18,31 @@
...
         {{ $clean_name := replaceRE "[ÉÈÊËĚÊ̌ΕĖ́Ę́É̩Έ]" "E" $clean_name }}
         {{ $clean_name := replaceRE "[F̌F́]" "F" $clean_name }}
-        {{ $clean_name := replaceRE "[ǦǴ]" "G" $clean_name }}
-        {{ $clean_name := replaceRE "[ȞH́Ή]" "H" $clean_name }}
+        {{ $clean_name := replaceRE "[ǦǴΓ]" "G" $clean_name }}
```
Comment (mal-tee) [inline layouts/company/list.html:26]:
```suggestion
        {{ $clean_name := replaceRE "[ǦǴ]" "G" $clean_name }}
```

File: layouts/company/list.html [hunk 8]
```diff
@@ -18,31 +18,31 @@
...
         {{ $clean_name := replaceRE "[ÌÍÎÏΊ]" "I" $clean_name }}
         {{ $clean_name := replaceRE "[J̌J́]" "J" $clean_name }}
-        {{ $clean_name := replaceRE "[ǨḰ]" "K" $clean_name }}
-        {{ $clean_name := replaceRE "[Ľ]" "L" $clean_name }}
-        {{ $clean_name := replaceRE "[M̌Ḿ]" "M" $clean_name }}
-        {{ $clean_name := replaceRE "[ÑŃ]" "N" $clean_name }}
-        {{ $clean_name := replaceRE "[ÒÓÔÕÖØΌ]" "O" $clean_name }}
-        {{ $clean_name := replaceRE "[P̌Ṕ]" "P" $clean_name }}
+        {{ $clean_name := replaceRE "[ǨḰΚ]" "K" $clean_name }}
+        {{ $clean_name := replaceRE "[ĽΛ]" "L" $clean_name }}
```
Comment (mal-tee) [inline layouts/company/list.html:31]:
```suggestion
        {{ $clean_name := replaceRE "[Ľ]" "L" $clean_name }}
```

**Review by mal-tee — CHANGES_REQUESTED** [review]
Thanks! 

I've left some final comments, I think these letters are too far of of the latin "counterparts" to group them together.

**Review by mal-tee — APPROVED** [review]
Thank you!

**Pull request merged**